aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
Thanks @Rick-Anderson and @blowdart

Currently the sample creates one seeded user who gets a role (which I will change to "ContactAdministrators"). The seeded user can do any operation because of ContactRoleAuthorizationHandler where we allow this one role. 

Then another user can be created in the sample by registering from the UI and that user can create contacts and do any operation but only on the contacts he creates because of ContactIsOwnerAuthorizationHandler.

You mention having a map of roles to operations, but we only have one role at the moment. Do we need a role per operation/permission? if so what would we name the roles since we don't want them to be named like permissions. And if we do that then I guess since there is no role management in the UI, we need to seed another user or perhaps a few users with specific roles so they can do one or more operations?